libcontainer/factory_linux.go

```diff
@@ -380,6 +381,24 @@ func (l *LinuxFactory) StartInitialization() (err error) {
 		return fmt.Errorf("unable to convert _LIBCONTAINER_LOGPIPE: %w", err)
 	}
 
+	// Get mount files (O_PATH)
+	envMountFileFds := os.Getenv("_LIBCONTAINER_MOUNT_FILE_FDS")
+	mountFilesArr := strings.Split(envMountFileFds, ";")
+	mountFiles := make([]*os.File, len(mountFilesArr))
+	for i, fdStr := range mountFilesArr {
```

Comment:
For setns init, there will be no `_LIBCONTAINER_MOUNT_FILE_FDS` env var, and yet you allocate a map (of zero len). This seems useless.

Reply:
Just have mountFiles as
```
var mountFiles []*os.File
fds := os.Getenv("_LIBCONTAINER_MOUNT_FILE_FDS")
if fds != "" {
	// allocate the map etc
	....
}	
```

Reply:
Cool, that is more or less the same. Will adjust to that later (quite late here), thanks!

Reply:
Done